ion/src/shared/events_keyboard.cpp

```diff
@@ -29,6 +29,13 @@ bool canRepeatEvent(Event e) {
 Event getEvent(int * timeout) {
```

Comment:
This function is called in the software running on the device. We try to avoid adding code that would be useful only for one platform when this code is built for all platforms. Indeed, we do not want to downgrade performance on the device to improve a functionality of the simulator.